sign/rpmgensig.c

```diff
@@ -238,11 +239,25 @@ static rpmtd makeSigTag(Header sigh, int ishdr, uint8_t *pkt, size_t pktlen)
     return sigtd;
 }
 
+pid_t gpgPid = 0;
```

Comment:
I don't trust myself to do a proper review this late on Friday afternoon (and neither should you) :smile:  but just a quick comment: this variable needs to be declared static, otherwise it'll end up visible in the library ABI.

Reply:
Indeed. No rush!

Declaration fixed.